themes/scalereal/static/css/style.css

```diff
@@ -3363,4 +3363,148 @@ lottie-player {
 
 .feedback-mobile  .owl-carousel .owl-dots {
   margin-top: -2px;
+}
+
+/* Client Success Stories Section */
+.client-success-stories-section .container .box {
+  padding: auto 98px ; 
+}
+.client-success-stories-section .box::before {
+  content: "";
+  position: absolute;
+  left: 47%;
+  top: 85px;
+  bottom: 85px; 
+  width: 2px;
+  height: 70%;
+  background-color: #cccbcb; 
+}
+.client-success-stories-section .container .box .heading.box-content{
+  margin:100px 25px auto auto;
+}
+.client-success-stories-section .container .box .heading.box-content h2{
+  padding-right: 86px;
+}
+.client-success-stories-section .container .box .box-content{
+  margin: 0px;
+  padding-left: 3px;
+  width: auto;
+}
+.client-success-stories-section .container .box .box-content .owl-one .owl-dots {
+  margin-top: 24px;
+  display: flex;
+  flex-direction: row; 
+  justify-content: center;
+}
+.client-success-stories-section .container .box .box-content p{
+  font-size: 22px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+  max-width: 423px;
+}
+.client-success-stories-section .container .box .box-content h3{
+  font-size: 26px;
+  padding: auto 25px 20px auto;
+}
+.client-success-stories-section .container .box .box-content img{
+  width: 100%; 
+}
+@media (min-width:767.23px) and (max-width:991.98px) {

```

Comment:
why specific px? Pls mention 768px and 992px

Reply:
the alginment of the row is only changing properly when i use 991.98px 
![image](https://github.com/scalereal/website/assets/76813100/02d76eeb-32c9-458a-b742-f6dffa8cfd7c)
![image](https://github.com/scalereal/website/assets/76813100/a9e42c6c-31c9-4688-aefe-88cb86773a75)
please have a look